test/functional/helpers/report-helper.js

```diff
@@ -0,0 +1,63 @@
+function checkTestRelevanceByBrowserAlias (test, browserAlias) {

```

Comment:
I vote for `filterError`. We can understand *filter by what* from the function arguments